CONTRIBUTING.md

```diff
@@ -1,27 +1,37 @@
 # Contributing to Round Manager and Grant Explorer
 
-Thank you for your desire to contribute to Gitcoin! Below you will find some ways to help, but please don't feel limited by them, and if you have any other ideas, pop by our [Discord](https://discord.gg/gitcoin)
+Thank you for your desire to contribute to Gitcoin! Below you will find some ways to help, but please don't feel limited
+by them, and if you have any other ideas, pop by our [Discord](https://discord.gg/gitcoin)
 
 ## Reporting issues
 
 Before reporting an issue, please make sure to verify the following:
+
 - You're running the latest version, including other components and systems the project might interact with
 - Your dependencies are up-to-date
 
-If the issue still persists, search the issue tracker. If there is an open ticket, please don't comment '+1' or similar - use the '👍' emoji. Only comment if you have additional context, reproduction instructions or anything that might help resolve the issue.
+If the issue still persists, search the issue tracker. If there is an open ticket, please don't comment '+1' or
+similar - use the '👍' emoji. Only comment if you have additional context, reproduction instructions or anything that
+might help resolve the issue.
 
 When opening a new issue, make sure it contains the following:
+
 - Step-by-step reproduction instructions
 - Screenshots or videos of the issue, if applicable
 - Version information of the project
 - Any other context that you feel would help resolve the issue
 
 ## Contributing code
 
-We welcome all code contributions, large or small. Please see [DEV.md](packages/round-manager/docs/DEV.md) for instructions on how to set up the project for development.
+We welcome all code contributions, large or small. Please see [DEV.md](docs/DEV.md) for
+instructions on how to set up the project for development.
 
-Before submitting a PR, please make sure that all tests, linting and formatting rules are passing.
+Before submitting a PR, please make sure that all the CI checks are passing. We also utilize git hooks for pre-commit
```

Comment:
In the interest of keeping instructions close to the place where they're needed, should this be part of the PR checklist?